src/retrace/retrace_worker.py

```diff
@@ -53,6 +53,10 @@ def begin_logging(self):
         if self.logging_handler is None:
             self.logging_handler = logging.FileHandler(
                 self.task._get_file_path(RetraceTask.LOG_FILE))
+
+        formatter = logging.Formatter(fmt="[%(asctime)s][%(levelname)-.1s] %(message)s",
```

Comment:
Hmm, maybe separate the time from the log level? A bit easier to scan that way, IMO.

Reply:
I mean adding some whitespace.

Reply:
> I mean adding some whitespace.

Done.